contents/handbook/growth/marketing/product-announcements.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Product announcements
+sidebar: Handbook
+showTitle: true
+---
+
+Marketing takes responsibility for coordinating and publicising PostHog news, including product announcements. 
+
+However, not all announcements require the same level of marketing support. We therefore group them into tiers which help us decide what level of investment is required from the Marketing team. Which tier applies is agreed upon by the Marketing and Product teams and agreement should be sought as early as possible if Marketing support is expected.
+
+### Tier 3: Service features
+Service features are those which introduce changes which we do not expect to have a noticeable impact for a majority of users.
+
+An example of a service feature would include bug fixes or minor performance improvements. 
+
+Marketing support for a service feature would typically include: 
+
+- [x] - Including the news in the PostHog newsletter.
+- [x] - Including the news in PostHog Array updates.
```

Comment:
PostHog Array is published by Product/Engineering anyways, so might be better to delegate to the release guide there?